lib/modules/manager/composer/schema.ts

```diff
@@ -304,6 +309,17 @@ export const ComposerExtract = z
 
         const gitRepo = gitRepos[depName];
         if (gitRepo) {
+          const bitbucketMatchGroups = bitbucketUrlRegex.exec(
+            gitRepo.url
+          )?.groups;
+
+          if (bitbucketMatchGroups) {
+            dep.datasource = BitbucketTagsDatasource.id;
```

Comment:
needs to be added to the supported datasources export inside `index.ts`

Reply:
I have no idea about the impact of that change, but I have updated as suggested.